header/p2p/subscriber.go

```diff
@@ -60,6 +60,11 @@ func (p *Subscriber) AddValidator(val header.Validator) error {
 	return p.pubsub.RegisterTopicValidator(PubSubTopic, pval)
 }
 
+// RemoveValidator removes pubsub message validation
+func (p *Subscriber) RemoveValidator() error {
```

Comment:
Uh, I think I had to add `RemoveValidator` in a separate PR but absence of it became actual only after adding the FraudService:
Validator in `header/p2p/subscriber`  is registering in `Syncer.Start`.  This validator is adding incoming Extheader to headerstore (please see `processIncoming`) under the hood. So, once BEFP is received we are removing validation and thus adding new Extheader to headerstore

Reply:
Just discussed with @vgonkivs how we can remove AddValidator completely.

Reply:
`P2PSubscriber` already has a Start and Stop methods. Stop for it already implements unregistering, complete stop of the topic and this what we need. We can expose those on the interface instead of new `RemoveValidator` and call it on Syncer, as Syncer already keeps the subscriber.

Reply:
Removed this functionality at all. Instead, Syncer will call exposed Stop method from P2PSubscriber